docs/applications/container-repositories/02-github.md

```diff
@@ -0,0 +1,133 @@
+| `IMAGE_NAME` | The image name of your build | variable | If it does not exist paired with the `NAMESPACE`, then it will be created for you |
+| `REGION` | The region where your cloud container repository is located | variable | Must be a valid cloud region. For example: `eastus` for Azure, or `us-west-1` for AWS. |
+
+## Workflow file
+
+To set up the GitHub Action, create a new file named `deploy.yaml` in the `.github/workflows` directory of your GitHub repository. You can use this workflow below as a starting point:
+
+```yaml title=".github/workflows/deploy.yaml"
+name: Deploy to Massdriver
+on:
+  push:
+    branches: [main]
+
+jobs:
+  push_and_deploy:
+    runs-on: ubuntu-latest
+    env:
+      MASSDRIVER_ORG_ID: ${{ secrets.MASSDRIVER_ORG_ID }}
+      MASSDRIVER_API_KEY: ${{ secrets.MASSDRIVER_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Massdriver CLI
+        uses: massdriver-cloud/actions@v3
+      - name: Push Image
+        uses: massdriver-cloud/actions/image_push@v3
+        with:
+          namespace: ${{ vars.NAMESPACE }}
+          image-name: ${{ vars.IMAGE_NAME }}
+          artifact: ${{ secrets.MASSDRIVER_ARTIFACT_ID }}
+          region: ${{ vars.REGION }}
+          image-tag: ${{ github.sha }}
+          build-context: ./
+      - name: Publish Bundle 
+        uses: massdriver-cloud/actions/bundle_publish@v3
+        with:
+          build-directory: ./
+      - name: Set Image Version 
+        uses: massdriver-cloud/actions/app_patch@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+          set: |
+            <insert-image-tag-path> = "${{ github.sha }}"
+      - name: Deploy App
+        uses: massdriver-cloud/actions/app_deploy@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+
+```
+
+You may need to modify the branch name in the `on: push:` section of the workflow to match the branch you're using for your application.
+
+When this GitHub Action runs, it will:
+* Build and push your image to your cloud container repository
+* Update the tag in your application package
+* Redeploy your application in Massdriver with the updated tag
+
+### Where can I find my secrets, project, target, and manifest names?
+
+* [Secrets & vars](#set-secrets-and-vars)
+* [Projects](/concepts/projects) are found on the Projects page of Massdriver
+* [Targets](/concepts/environments) are found listed inside your project
+* [Manifests](/concepts/manifests) are the abbreviated name of your bundle or application
+
+![Manifest Name](manifest.png)
+
+### Where can I find my image tag path?
+
+The image tag path is the path configured within your application's `massdriver.yaml` file. For example, if your `massdriver.yaml` file looks like this:
```

Comment:
```suggestion
The image tag path is the [`jq`](https://stedolan.github.io/jq/manual/) formatted path to the image tag field in the params section of your application's `massdriver.yaml` file. For example, if your `massdriver.yaml` file looks like this:
```

Slight change - wanted to make sure we get `jq` in there and make sure they understand its in the `params` section. You can reword if you have a better way of including that.

Reply:
I like this wording, copy/pasted